clang/test/CodeGenCXX/dynamic-cast-nonzero-default-address-space.cpp

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -I%S %s -triple spirv64-unknown-unknown -fsycl-is-device -emit-llvm -fcxx-exceptions -fexceptions -o - | FileCheck %s
```

Comment:
Could you use update_cc_test_checks --check-globals here to just check the whole function?